### PR TITLE
got rid of By thing, reduce sleep to 1 sec

### PR DESCRIPTION
**Pull request by AndrewKorchevskyi**
easy

### CODE REVIEW

File: src/main/java/pages/ShopPage.java [hunk 1]
```diff
@@ -19,8 +21,11 @@ public ShopPage(WebDriver driver) {
     @FindBy(xpath = ".//*[@id='cart-widget-button']")
     private WebElement cartWidgetBtn;
 
-    private By getItemByVisibleText(String itemVisibleText) {
-        return By.xpath(String.format(".//*[@class='product-details']//h3[contains(text(), \"%s\")]", itemVisibleText));
+    @FindBy(xpath = ".//*[@class='product-details']//h3")
+    private List<WebElement> shopItems;
+
+    private WebElement getItemByVisibleText(String itemVisibleText) {
+        return shopItems.stream().filter(webElement -> webElement.getText().equals(itemVisibleText)).findAny().orElse(null);
```
[the remark this answers is not in the document]
Comment (AndrewKorchevskyi) [reply src/main/java/pages/ShopPage.java]: No, learn

[the remark this answers is not in the document]
Comment (AndrewKorchevskyi) [reply src/main/java/pages/ShopPage.java]: You already have lambdas in project, remove them too

[the remark this answers is not in the document]
Comment (AndrewKorchevskyi) [reply src/main/java/pages/ShopPage.java]: shakh i mat suka

[the remark this answers is not in the document]
Comment (artemmlg) [reply src/main/java/pages/ShopPage.java]: Already removed lambda expressions from moi project, haven't pushed already

File: src/main/java/pages/BasePage.java [hunk 2]
```diff
@@ -72,17 +62,9 @@ public void switchIntoFrame(WebElement frameElement) {
...
-
-    public int getSize(By by) {
-        return getWebElements(by).size();
-    }
-
-    public void moveToElementFromList(By by, int num) {
+    public void moveToElementFromList(List<WebElement> elements, int num) {
         Actions actions = new Actions(driver);
-        WebElement element = getWebElements(by).get(num);
+        WebElement element = elements.get(num);
```
[the remark this answers is not in the document]
Comment (artemmlg) [reply src/main/java/pages/BasePage.java]: Prost

File: src/main/java/pages/ShopPage.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (AndrewKorchevskyi) [reply src/main/java/pages/ShopPage.java]: ne budu, tut eto horosho zahodit

**Pull request merged**